index.js

```diff
@@ -1,14 +1,36 @@
 const express = require('express')
+const bodyParser = require('body-parser')
+const { getAllMonths, getMonthByName, getMonthBySlug } = require('./controllers/months')
+const { getAllMoons, getMoonByName, getMoonByMonth } = require('./controllers/moons')
+const {
+  getAllActivities, getActivitiesByMonth,
+  getActivitiesByMoon, addActivities, deleteActivities
+} = require('./controllers/activities')
+
 
 const app = express()
 
 app.set('view engine', 'pug')
 
 app.use(express.static('public'))
 
-app.get('/months', (request, response) => {
-  return response.render('index')
-})
+app.use(bodyParser.json())
+
+app.get('/', (request, response) => response.render('index'))
+
+app.get('/months', getAllMonths)
+app.get('/months/:name', getMonthByName)
+app.get('/months/:slug', getMonthBySlug)
```

Comment:
This is the same route. When the user types `/months/something` Express has no way of understanding if they intended it as a name or a slug. All it can do is match on `/months/` and place the `something` into the variable `name` since that is what you told it to do. No matter what the user types after `/months/` it will match the route on line 22 and place anything they typed into the `name` variable. You're expecting Express to understand the user's intentions here.